media/js/firefox/installer-help-experiment.js

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function (Mozilla) {
+    'use strict';
+
+    /* update dataLayer with experiment info */
+    var href = window.location.href;
+    if (href.indexOf('v=') !== -1) {
+        if (href.indexOf('v=a') !== -1) {
+            window.dataLayer.push({
+                'data-ex-variant': 'installer-help-control',
+                'data-ex-name': 'CRO-Installer-Help-Experiment'
+            });
+        } else if (href.indexOf('v=b') !== -1) {
+            window.dataLayer.push({
+                'data-ex-variant': 'installer-help-v1',
```

Comment:
Nit: should this be `vb` to reflect `v=b` in the URL?

Reply:
I collaborated with Saptarshi and he requested this value. 👍